.circleci/config.yml

```diff
@@ -7,6 +7,8 @@ jobs:
         environment:
           NODE_ENV: test
           TARGET: CANARY
+          DETECT_CHROMEDRIVER_VERSION: 'false'
```

Comment:
See https://www.npmjs.com/package/chromedriver#detect-chromedriver-version.
The doc doesn't say it, but by default this feature is enabled. I had to disable it in order to make it use `CHROMEDRIVER_FILEPATH`